code/week5/animal.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <iostream>
+
+class Animal {
+	std::string name;
```

Comment:
I, personally, am not a fan of the implied access modifiers (the fact that the members declared before "public:" here default to private). At the end of the day, this is just a style choice, but I prefer to explicitly state "private:" so intentions are clear. This also allows you to move private member declarations to the end of the class declaration, moving your public interface at the top. This is nice as most people reading your header will be primarily interested in the public interface of your class.